src/script/components/DeviceCard.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import cx from 'classnames';
+import {ClientClassification} from '@wireapp/api-client/src/client';
+import {t} from '../util/LocalizerUtil';
+import {registerReactComponent} from 'Util/ComponentUtil';
+
+import type {ClientEntity} from '../client/ClientEntity';
+import NamedIcon from './NamedIcon';
+import LegalHoldDot from './LegalHoldDot';
+
+export interface DeviceCardProps {
+  click?: (device: ClientEntity) => void;
+  device: ClientEntity;
+  showIcon?: boolean;
+  showVerified?: boolean;
+}
+
+const DeviceCard: React.FC<DeviceCardProps> = ({
+  click,
```

Comment:
Let's specify a noop click as default parameter here (noop exists in some util already)

Reply:
doing a noop here will break a part of this component that is rendering based on existence of this prop (that `noop` is a true value)
![image](https://user-images.githubusercontent.com/63250054/106922322-2f986e80-6722-11eb-87d0-743f107e6397.png)

Reply:
Or you could check like this:
```
const clickable = click !== noop;
```